netci.groovy

```diff
@@ -1853,12 +1883,17 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
                 contextString += " Build and Test"
                 triggerString += "\\W+Build and Test"
             }
+            
+            if(isJitStressScenario(scenario)) {
+                def displayStr = getStressModeDisplayName(scenario)
+                contextString += "(Jit - ${displayStr})"
+            }
 
             triggerString += ".*"
 
             switch (os) {
                 case 'Ubuntu':
-                    if (scenario == 'innerloop') {
+                    if (scenario == 'innerloop' || scenario == 'no_tiered_compilation_pri0') {
```

Comment:
Should the switch for `'Windows_NT'` below also include `'no_tiered_compilation_pri0'`?

Reply:
The policy we wanted to go with was Ubuntu-arm + Windows-x86 and Windows-x64, so that leaving out Windows-arm was by design. I'm not opposed to changing the policy as needed though.